closed/openssl-conf/semeru.conf

```diff
@@ -0,0 +1,45 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2025, 2025 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+# ===========================================================================
+
+my %targets = (
+	"semeru"            => { shlib_variant => "-semeru", template => 1 },
+	"semeru-linux"      => {
+		inherit_from        => [ "semeru" ],
+		shared_extension    => ".so",
+		shared_ldflag       => add("-Wl,-rpath,'\$\$'ORIGIN"),
+		template            => 1
+	},
+
+	"semeru-ap64"       => { inherit_from => [ "aix64-cc" ] },
+	"semeru-ap64-clang" => { inherit_from => [ "aix64-clang" ] },
+	"semeru-oa64"       => { inherit_from => [ "darwin64-x86_64-cc" ] },
+	"semeru-or64"       => { inherit_from => [ "darwin64-arm64-cc" ] },
+	# The name must start with "VC-", otherwise Windows builds fail.
+	"VC-wa64"           => { inherit_from => [ "VC-WIN64A" ] },
+    "VC-wi32"           => { inherit_from => [ "VC-WIN32" ] },
+    "semeru-ap32"       => { inherit_from => [ "aix32-cc" ] },
+	"semeru-xi32"       => { inherit_from => [ "semeru-linux", "linux-x86" ] },
+	"semeru-xz31"       => { inherit_from => [ "semeru-linux", "linux32-s390x" ] },
+	"semeru-xa64"       => { inherit_from => [ "semeru-linux", "linux-x86_64" ] },
```

Comment:
Please indent only with tabs and reorganize the list so the names are (mostly) sorted (the VC-* ones can stay where semeru-w* would go).